### ISSUE TITLE
Pylint & Flake8: Error codes -> string comments

### ISSUE THREAD

**Issue by Bibo-Joshi** [python-telegram-bot/python-telegram-bot#2664]
e.g. 

```
# pylint: disable=line-too-long
```
rather than

```
# pylint: disable=C0301
```

IMO this has low priority, but it may improve readability a bit - even though I fear that it will introduce a bunch of artificial line breaks. But whatever :D

**Comment by areeb1501** [comment]
Hi Bibo,
So do we need to change all the codes to their messages?

**Comment by Bibo-Joshi** [comment]
Yes, that would be the idea.

**Comment by areeb1501** [comment]
Oh okay, so do I find instances of all such pylint messages and replace them with the text mentioned in the docs?

Currently, it is written like this.
`python
disable = C0330,R0801,R0913,R0904,R0903,R0902,W0511,C0116,C0115,W0703,R0914,R0914,C0302,R0912,R0915,R0401
`

Do I modify the code to look something like this?
`python
disable = bad-continuation, duplicate-code, too-many-arguments, too-many-public-methods, too-few-public-methods, too-many-instance-attributes, fixme, missing-function-docstring, missing-class-docstring, broad-except, too-many-locals, too-many-locals, too-many-lines, too-many-branches, too-many-statements, cyclic-import
`

**Comment by Bibo-Joshi** [comment]
Yes, that's what we had in mind.

If you want to PR for this, please make sure to base your branch on v14. see https://github.com/python-telegram-bot/python-telegram-bot/issues/2665#issuecomment-922893897

BTW, regarding the hacktoberfest label: AFAIK only PRs opened in octobar are eligible for hacktoberfest, so if you're interesetd in PRin for this and want it tou count for hacktoberfest, you should probably wait until next Friday with opening a PR :)

**Comment by areeb1501** [comment]
Okay cool! 
I'll open my pr, next friday so it counts towards the Hactoberfest. 

Also, i did not find any other instances of the pylint codes, apart from this. Is there any other place where this kind of modification is required?

**Comment by Bibo-Joshi** [comment]
Yes, lot's of places: https://github.com/python-telegram-bot/python-telegram-bot/search?q=%23+pylint%3Adisable

**Comment by areeb1501** [comment]
Thanks, i'll look into these files! 🚀

**Comment by areeb1501** [comment]
Can I instead PR right away? I have made the changes in the required files as well.

**Comment by areeb1501** [comment]
> Can I instead PR right away? I have made the changes in the required files as well.

Friday is a really long time. I'll find other issues to work on for Hacktoberfest. Thanks for letting me know anyways. :)

**Comment by Bibo-Joshi** [comment]
Sure, go ahead. Was just pointing it out to prevent disappointment ;) However, I can't give a promise on how quick the review process will be ;)

**Comment by areeb1501** [comment]
Thanks a lot!!
Is there any way that you can verify the changes I've made and also make it count towards Hacktoberfest?

**Comment by Bibo-Joshi** [comment]
we can go through the review process, close the PR and then you PR again in october, but tbh I'd prefer for you to just wait the 9 days if you want the PR to count for HF …

**Comment by areeb1501** [comment]
Okay, I'll wait then. If thats what you think is good. Because I'm new to OS, and am still not really familiar with the processes. So i'll stick to what you say. 😀 🚀 

Cheers!

**Comment by Bibo-Joshi** [comment]
Closing, as #2700 solves this for pylint and apparently I was misinformed about flake8 having a similar functionality …
